rero_ils/accounts_views.py

```diff
@@ -46,6 +51,35 @@ def user_exists(email):
         raise ValidationError(_('INVALID_USER_OR_PASSWORD'))
 
 
+def validate_password(password):
+    """Validate the password."""
+    length = current_app.config.get('RERO_ILS_PASSWORD_MIN_LENGTH', 8)
+    special_char = current_app.config.get('RERO_ILS_PASSWORD_SPECIAL_CHAR')
+    if len(password) < length:
+        raise ValidationError(
+            _('Field must be at least %(min)d characters long.', min=length))
+    if not password_validator(password, length=length,
+                              special_char=special_char):
+        messages = {
+            'invalid': _('The password must contain at least one upper'
+                         ' case letter, one lower case letter'
+                         ' and one number.'),
+            'invalid_special_char': _('The password must contain at least'
+                                      ' one upper case letter, one lower'
+                                      ' case letter, one number and one'
+                                      ' special character.')
+        }
+        message = messages['invalid_special_char'] if special_char \
+            else messages['invalid']
+        raise ValidationError(message)
```

Comment:
Didn't you already create a validation password method ? Why don't use it .... I think I miss something :s